deploy/vagrant/conf/gce.yml.template

```diff
@@ -0,0 +1,73 @@
+# Setting up your Google Cloud Project
+#
+# The Project ID for your Google Cloud Platform account.
+Google_Cloud_Project_ID:
+
+# The Client Email address for your Service Account.
+Service_Acount_Email_Address:
+
+# The location of the JSON private key file matching your Service Account.
+json_Key_Location: 
+
+
+#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
+#
+# This is the path to SSH keys. The easiest way to setup GCE keys is using the 
+# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to 
+# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
+# directly or manually manage keys.
+Key_Path: "~/.ssh/google_compute_engine"
+
+# This is the username associated with the keys above.
+SSH_Username: "gce-user"
+
+
+# Configuring virtual machines
+#
+# Preemptible VMs are short lived, low cost, ephemeral VMs. They run for 24 hours or less
+# but my be preempted at any time. Otherwise they are exactly they same as standard VMs.
+# They are ideal for batch jobs and fault-tolerant workloads, assuming your application is
+# can withstand possible instance preemptions.
+# See documentation (https://cloud.google.com/compute/docs/instances/preemptible) for more details.
+# Boolean whether to enable preemptibility. Default is false.
+Preemptible: false
+
+# See the web docs (https://cloud.google.com/compute/docs/machine-types) or 
+# CLI (gcloud compute machine-types list) for a list of available machine types.
+# When using spark, we recommend "n1-highmem-2" as compiling spark requires at least 2G of memory.
+Machine_Type: "n1-standard-1"
+
+# Scopes
+# An array of OAuth2 account scopes for services that the instance will have access to. 
+# Those can be both full API scopes, just endpoint aliases (the part after ...auth/), and gcloud utility aliases.
+# Examples include: ['storage-full', 'bigquery', 'https://www.googleapis.com/auth/compute'].
+# For simplicity, ['https://www.googleapis.com/auth/cloud-platform'] provides access to all resources.
+# See documentation on scopes generally (https://cloud.google.com/docs/authentication#oauth_scopes) 
+# and specific to GCE (https://cloud.google.com/compute/docs/authentication#using).
+Scopes: ['https://www.googleapis.com/auth/cloud-platform']
+
+# The disk size in GB. The default is 10.
```

Comment:
Is this referring to Persistent Disk size? can you please also add a link for it?